### DISCUSSION TITLE
win predictions are never 50-50

### DISCUSSION THREAD

**Discussion by mvela-king** [OpenDebates/openskill.py#113] [Q&A] [answered]
I have written a small program to show what seems to be a problem with win prediction in PlackettLuce model. When running this code I expected to get a prediction of precisely [0.5,0.5], since both teams are the same size and all players have the same rating, but that is never the case. For a team size of 2 the values I get are: [0.44334259670042503, 0.556657403299575], and for other team size, these values change but are never the expected [0.5,0.5]. Is this a bug, or is there an explanation for this difference?

```python
from openskill.models import PlackettLuce

model = PlackettLuce()
team_size = 2
teamA = []
teamB = []

for i in range(team_size):
    name = 'pA'+ str(i)
    rating = model.create_rating([25.0, 25/3], name)
    teamA.append(rating)
    name = 'pB'+ str(i)
    rating = model.create_rating([25.0, 25/3], name)
    teamB.append(rating)

match = [teamA,teamB]
prediction = model.predict_win(match)
print(prediction)
```

**Comment by vivekjoshy** [comment, marked as answer]
This was a bug introduced in v5.0.0. It was fixed in v5.0.1 and later.

Here is a screenshot with up to 20 players per team:

![image](https://github.com/OpenDebates/openskill.py/assets/8206808/6116b020-a857-474c-a54d-33cc7587ffc6)
